src/bindingHandlers/delayedInput.ts

```diff
@@ -0,0 +1,24 @@
+import * as ko from "knockout";
+
+ko.bindingHandlers["delayedInput"] = {
+    init: (element: HTMLElement, valueAccessor: () => ko.Observable<string>): void => {
+        const inputObservable: ko.Observable = valueAccessor();
+
+        let timeout;
+
+        const scheduleUpdate = (value: string) => {
```

Comment:
can we use knockout rateLimit extender instead of this?
https://knockoutjs.com/documentation/rateLimit-observable.html

Reply:
Replaced with rateLimit extended.